tests/compatibility-test.py

```diff
@@ -45,8 +46,10 @@ def apply_kuberay_resources():
     shell_assert_success('kind load docker-image kuberay/apiserver:{}'.format(kuberay_sha))
     shell_assert_success(
         'kubectl create -k manifests/cluster-scope-resources')
+    # use kustomize to build the yaml, then change the image to the one we want to testing.
     shell_assert_success(
-        'kubectl create -k manifests/base')
+        'kustomize build manifests/base | sed "s/image: kuberay\\/\\(.*\\):nightly/image: kuberay\\/\\1:{}/g" | kubectl apply -f -'.format(
```

Comment:
Can we use command like below? @wilsonwang371 

```
kubectl kustomize edit set image kuberay/operator=kuberay/operatorr:commit_hash
kubectl apply -k manifests/bases

```

Reply:
done